### PR TITLE
chore: restore revapi

### PR DESCRIPTION
**Pull request by monperrus**
Revapi was executed by an Inria server, now dead, it is now executed by Travis.

### CODE REVIEW

File: src/main/java/spoon/reflect/visitor/ImportCleaner.java [hunk 1]
```diff
@@ -118,7 +118,7 @@ protected void handleTypeReference(CtTypeReference<?> reference, Context context
 		}
 	}
```
Comment (nharrand) [inline src/main/java/spoon/reflect/visitor/ImportCleaner.java]: If these inner classes become public, shouldn't them be documented?

Comment (monperrus) [reply src/main/java/spoon/reflect/visitor/ImportCleaner.java]: done!

**Comment by nharrand** [comment]
Ready for merge?

**Comment by monperrus** [comment]
yes, it works!

**Pull request merged**